codepipeline-tf/modules/xc3_codepipeline/readme.md

```diff
@@ -0,0 +1 @@
+This pipeline has currently different stages to Source, Build, Deploy and destroy the code.
```

Comment:
Add details on the setup and other relevant things as well.

Reply:
sure